Error building on 32-bit platforms

Firstly, thank you for this project!

I tried to build the latest version for a 32-bit machine and got the following error:

```
$ GOARCH=386 GOOS=linux go build ./...
# github.com/stangelandcl/ppmd
$HOME/pkg/mod/github.com/stangelandcl/ppmd@v0.1.0/reader.go:25:42: (0xFFFFFFFF - 12 * 3) (untyped int constant 4294967259) overflows int
```

I know the error is coming from https://github.com/stangelandcl/ppmd, but that project has not been updated in four years, so I figured you'd want to know :)

--- a/.golangci.yaml
+++ b/.golangci.yaml
@@ -39,7 +39,7 @@ linters:
     - gocyclo
     - godot
     - goheader
-    - gomoddirectives
+    #- gomoddirectives
     - gomodguard
     - goprintffuncname
     - gosec

--- a/go.mod
+++ b/go.mod
@@ -18,6 +18,8 @@ require (
 	golang.org/x/text v0.37.0
 )
 
+replace github.com/stangelandcl/ppmd v0.1.0 => github.com/bodgit/ppmd v0.0.0-20260522230604-9b76e8bd7ee8
+
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect

--- a/types.go
+++ b/types.go
@@ -34,13 +34,13 @@ const (
 	idNumUnpackStream
 	idEmptyStream
 	idEmptyFile
-	idAnti //nolint:deadcode,varcheck
+	idAnti
 	idName
 	idCTime
 	idATime
 	idMTime
 	idWinAttributes
-	idComment //nolint:deadcode,varcheck
+	idComment
 	idEncodedHeader
 	idStartPos
 	idDummy

